test/perf/skipped_describe.test.ts

```diff
@@ -0,0 +1,11 @@
+import {bench, describe} from "../../src/index.js";
```

Comment:
I'm not sure how to review this.  I don't get what this file is for from the comment.